demos/DubinsAirplane.py

```diff
@@ -129,6 +129,29 @@ def plan(space : str, planner : str):
             print("Approximate solution. Distance to goal is ", ss.getProblemDefinition().getSolutionDifference())
         print("Path length is ", length)
 
+        if path.getStateCount() > 1:
+            for idx in range(path.getStateCount() - 1):
+                from_state = path.getState(idx)
+                to_state = path.getState(idx + 1)
+                space = ss.getStateSpace()
+                path_type = space.getPath(from_state, to_state)
+                db_path = path_type.path_
+
+                print(f"from_state: {from_state[0]:.1f}, {from_state[1]:.1f}, "
```

Comment:
This is a nice demonstration that the Python bindings work, but it is only correct for OwenStateSpace. It doesn't work for the other types.  I'd leave the whole `if path.getStateCount() > 1: ...` block out

Reply:
Updated in https://github.com/ompl/ompl/pull/1261/commits/19523f683c6f2d1a18bb05b1a39e0f698d45c3d2